src/components/Species/index.tsx

```diff
@@ -512,7 +512,7 @@ export default function SpeciesList({ organization, reloadData, species }: Speci
                   setSelectedRows={isUserContributor ? undefined : setSelectedSpeciesRows}
                   showTopBar={true}
                   Renderer={SpeciesCellRenderer}
-                  onSelect={selectAndEditSpecies}
+                  onSelect={!isUserContributor ? selectAndEditSpecies : undefined}
```

Comment:
i made a mistake by calling this `isUserContributor` - we should really have a flag that says `userCanEdit`.. we can change it later i guess

Reply:
I will change it